aws/resource_aws_ses_event_destination.go

```diff
@@ -53,20 +53,21 @@ func resourceAwsSesEventDestination() *schema.Resource {
 				Type:          schema.TypeSet,
 				Optional:      true,
 				ForceNew:      true,
-				ConflictsWith: []string{"kinesis_destination"},
+				MaxItems:      1,
```

Comment:
Interestingly enough, this resource actually did appear to support multiple `cloudwatch_destination` previously, even if it wasn't appropriately acceptance tested. This change will likely be reverted in #6690